bc_obps/registration/api/user_operator.py

```diff
@@ -78,11 +78,13 @@ def save_operator(payload: UserOperatorOperatorIn, operator_instance: Operator,
             "mailing_address_id",
             "website",
             "business_structure",
+            "cra_business_number",
+            "bc_corporate_registry_number",
```

Comment:
If we updated all fields, we wouldn't have to remember to update this list every time we add a field (I can see this bug happening again). Was `operator_related_fields` for optimization?

Reply:
Yes, I want to pass this list to 'update_fields' to optimize the save method. However, I also want to prevent unintended field updates in the 'save_operator' function if we pass different data using the payload.